corehq/apps/app_manager/static/app_manager/js/detail-screen-config.js

```diff
@@ -331,6 +331,13 @@ var DetailScreenConfig = (function () {
                 {label: "Case", value: "case"}
             ]).val(this.original.model);
             this.field = uiElement.input().val(this.original.field).setIcon(icon);
+
+            // Make it possible to observe changes to this.field :
+            this.field.observableVal = ko.observable(this.field.val());
+            this.field.on("change", function(){
+                that.field.observableVal(that.field.val());
+            });
```

Comment:
This looks like it's one-way. What happens if you set `observableVal`? Maybe that doesn't matter for your own change, but someone else might try and use it somewhere and that'll be really confusing. Since this is a short term solution, I'd just make a comment saying that it's one-way, and that using `observableVal` extensively is not recommended.

Eventually we'll get rid of this `uiElement` business altogether and and replace it with observable values and knockout templates as I've already started doing in phases (like a month ago).